modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import getRandomString from '../../utils/getRandomString';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analyticsSettings';
+import getFragmentDefinition from '../layout-content-page-editor-web/utils/getFragmentDefinition';
+import getPageDefinition from '../layout-content-page-editor-web/utils/getPageDefinition';
+import {faroConfig} from './faro.config';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+const createSitePage = async function (apiHelpers, pageTitle) {
+	const company = await apiHelpers.jsonWebServicesCompany.getCompanyByWebId(
+		'liferay.com'
+	);
+
+	const group = await apiHelpers.jsonWebServicesGroup.getGroupByKey(
+		company.companyId,
+		'Guest'
+	);
+
+	return await apiHelpers.headlessDelivery.createSitePage({
+		pageDefinition: getPageDefinition([
+			getFragmentDefinition(getRandomString(), 'BASIC_COMPONENT-heading'),
+		]),
+		siteId: group.groupId,
+		title: pageTitle,
+	});
+};
+
+const goToWithReferrer = async function (page, referrer, url) {
+	await page.goto(referrer);
+
+	await page.evaluate((url) => {
+		const aTag = document.createElement('a');
+
+		aTag.href = url;
+
+		aTag.click();
+	}, url);
+};
+
+test('shows outside pages in path analysis', async ({apiHelpers, page}) => {
+	const pageTitle = 'My Page';
+
+	const sitePage = await createSitePage(apiHelpers, pageTitle);
+
+	await syncAnalyticsCloud(page);
+
+	await goToWithReferrer(
+		page,
+		'https://www.google.com',
+		liferayConfig.environment.baseUrl
+	);
+
+	await page.waitForTimeout(10000);
+
+	await page.getByText(pageTitle).first().click();
+
+	await page.waitForTimeout(10000);
+
+	await page.goto(faroConfig.environment.baseUrl);
+
+	await page
+		.getByRole('link', {
+			name: 'FARO-DEV-liferay Liferay Demo Enterprise Plan',
+		})
+		.click();
+
+	await page.locator('.channels-menu.button-root').click();
+
+	await page.locator('.dropdown-item.sites-dropdown-item').last().click();
```

Comment:
Hey @marcosapmf, I think you could get the name of the property that you will use in your tests before executing the `await syncSite(page);` in the `syncAnalyticsCloud` function. This way, you will know the name of the property that you will need to search for and select in the dropdown.